src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import core.basesyntax.Figure;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(1,6)) {
+            case 1: return new Circle(randomDouble(1, 10), ColorSupplier.getRandomColor());
```

Comment:
replace switch statement with switch expression, remove return null, it's a bad practice